client/grpc/tmservice/service.go

```diff
@@ -105,10 +105,14 @@ func (s queryServer) GetLatestValidatorSet(ctx context.Context, req *GetLatestVa
 	}
 
 	for i, validator := range validatorsRes.Validators {
+		anyPub, err := codectypes.NewAnyWithValue(validator.PubKey)
+		if err != nil {
+			return nil, err
+		}
 		outputValidatorsRes.Validators[i] = &Validator{
 			Address:          validator.Address,
 			ProposerPriority: validator.ProposerPriority,
-			PubKey:           validator.PubKey,
+			PubKey:           anyPub,
```

Comment:
Could you add a test (e.g. in TestQueryLatestValidatorSet) to make sure this works? I guess like a `validatorSetRes.Validators[0].PubKey.GetCachedValue() == val.PubKey` (or something like that)